LayoutTests/platform/mac/editing/execCommand/find-after-replace-expected.txt

```diff
@@ -29,8 +29,6 @@ layer at (0,0) size 800x600
             RenderBlock {HTML} at (0,0) size 300x150
               RenderBody {BODY} at (8,8) size 284x134 [bgcolor=#FFFFE0]
                 RenderText {#text} at (0,0) size 55x18
-                  text run at (0,0) width 26: "A B"
-                  text run at (25,0) width 5: " "
-                  text run at (29,0) width 26: "A B"
+                  text run at (0,0) width 55: "A B A B"
```

Comment:
Why does this change?

Reply:
The progression was caused by some other recent code change, not in this pull request. We can land this separately.